modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -18,12 +18,6 @@ The following options are used for {rh-openstack} CCM authentication with the {r
 |===
 | Option | Description
 
-| `auth-url`
-| The {rh-openstack} Identity service URL. For example, `http://128.110.154.166/identity`.
-
-| `os-endpoint-type`
-| Optional. The type of endpoint to use from the service catalog. If unset, public endpoints are used.
-
 | `ca-file`
```

Comment:
I actually think every one of these options in the general section is not used by us. They all look like things that would belong in a `cloud.yaml` file and therefore they are things the user should set there (since we're using `clouds.yaml` file), not here. @pierreprinetti @MaysaMacedo wdyt?

Reply:
Good point, Stephen. I would say those are not needed given that we enforce to use cloud.yaml